shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/general/MySQLGeneralScalingIT.java

```diff
@@ -91,5 +98,6 @@ public void assertManualScalingSuccess() throws InterruptedException {
         assertPreviewTableSuccess("t_order_item", Arrays.asList("ds_2", "ds_3", "ds_4"));
         restoreScalingSourceWriting(jobId);
         assertRestoreScalingSourceWriting();
+        getJdbcTemplate().execute(String.format("DROP SCALING %s", jobId));
```

Comment:
If we need to drop job when it's completed, then it need to `DROP SCALING` even job fail.

Could we keep the job after it's completed, and clean it on startup?